app/scripts/controllers/mmi-controller.test.ts

```diff
@@ -813,7 +812,7 @@ describe('MMIController', function () {
 
   describe('setAccountAndNetwork', () => {
     it('should set a new selected account if the selectedAddress and the address from the arguments is different', async () => {
-      const selectedAccountSpy = jest.spyOn(controllerMessenger, 'call');
+      const selectedAccountSpy = jest.spyOn(ControllerMessenger.prototype, 'call');
```

Comment:
Instead of spying on all instances of ControllerMessenger, can we use `controllerMessenger` instead? I think that would be a lot more straightforward. If this is not accessible can we make it accessible for ourselves?